docs/source/usersguide/input.rst

```diff
@@ -374,37 +374,98 @@ OpenMOC does not place a limit on the hierarchical depth - or number of nested u
+This type of discretization is particularly useful for codes which can make use of an `unstructured mesh`_, such as OpenMOC with its general :ref:`Constructive Solid Geometry <constructive_solid_geometry>` formulation. To subdivide cylindrical fuel pins into rings and sectors in an LWR model would require a substantial amount of work for the user to create the necessary ``ZCylinder`` and/or ``Plane`` objects. Since this is a commonly needed feature for many users, OpenMOC includes the ability to automatically subdivide pin cells of cylindrical fuel pins into equal volume rings and equally spaced angular sectors. In particular, OpenMOC uses **cell cloning** to create clones (or copies) of a ``Cell`` object and differentiates each one with ``ZCylinder`` or ``Plane`` objects to subdivide the pin cell.
 
-The following code snippet illustrates how a user may designate a positive integral number of rings and sectors for fuel pin and moderator ``Cells`` using the ``Cell.setNumRings(...)`` and ``Cell.setNumSectors(...)`` class methods.
+Rings can be created inside of a ``ZCylinder``, between two ``ZCylinder`` objects, or outside of a ``ZCylinder``. If rings are created outside of a ``ZCylinder`` and the exterior bounding surface is the ``Cell`` boundaries, then an effective bounding cylinder radius :math:`R_\textit{eff}` is calculated as
+
+.. math::
+  R_\textit{eff} = \sqrt{\frac{\Delta x \Delta y}{\pi}}
+
+where :math:`\Delta x` is the width of the ``Cell`` in the :math:`x`-direction and :math:`\Delta y` is the width of the ``Cell`` in the :math:`y`-direction. Equal volume rings are then created between the inner bounding ``ZCylinder`` and this effective bounding radius. Note that in this case the regions will not be of equal volume since the effective bounding cricle will lay partially outside of the boundaries of the ``Cell``.
+
+The following code snippet illustrates how a user may designate a positive integral number of rings and sectors for fuel pin ``Cells`` and a positive integral number of sectors with no rings for moderator ``Cells`` using the ``Cell.setNumRings(...)`` and ``Cell.setNumSectors(...)`` class methods.
 
 .. code-block:: python
 
-    # Subdivide the fuel region into 3 rings and 8 angular sectors
+    # Subdivide the fuel region into 3 rings and 12 angular sectors
     fuel.setNumRings(3)
-    fuel.setNumSectors(8)
+    fuel.setNumSectors(12)
 
-    # Subdivide the moderator region into 8 angular sectors
+    # Subdivide the moderator region into 16 angular sectors
+    moderator.setNumSectors(16)
+
+The plots shown below illustrate the pin cell material layout (left) and flat source region layout (right) where the flat source regions have been discretized using 3 equal volume rings and 12 sectors in the fuel and 16 sectors in the moderator.
```

Comment:
is the figure below really materials on the left and fsrs on the right? The left file is "pin-cell-fsrs-1.png" and the right is "pin-cell-fsrs-2.png" which indicates to me that this is really showing two different FSR discretizations. If that isn't the case, then perhaps change the filenames to be more clear what is being shown.